core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -533,6 +542,9 @@ public CHILD_CLASS merge(final View view) {
                 if (null != view.config) {
                     getThisView().config.putAll(view.config);
                 }
+
+                getThisView().setAllEntities(view.allEntities);
```

Comment:
Maybe it would be better to decouple the clone from the fromJson / toJson methods and have a separate tests for this?

Reply:
After discussion decided to leave the clone method as is